spec/support/features/session_helper.rb

```diff
@@ -182,6 +182,18 @@ def sign_in_user(user = create(:user), email = nil)
       user
     end
 
+    def sign_in_user_with_eligible_platform_auth_available(
+      user = create(:user), email = nil
+    )
+      email ||= user.email_addresses.first.email
+      password = user.password
+      allow(UserMailer).to receive(:new_device_sign_in).and_call_original
+      visit new_user_session_path
+      set_hidden_field('platform_authenticator_available', 'true')
```

Comment:
Per some of the build failures, maybe we should set this as the default behavior for signing in in the existing spec helper(s)?